### PR TITLE
Extract interactable sensors from robot visualizer.

### PR DESCRIPTION
**Pull request by calvertdw**

### CODE REVIEW

File: ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2InteractableSensors.java
Comment (ds58) [inline ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2InteractableSensors.java:1]: There's probably not a good design fix right now. But, all of this is robot specific imo.

Comment (calvertdw) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2InteractableSensors.java:1]: It's a step in the right direction, though, right?

Comment (TomaszTB) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2InteractableSensors.java:1]: I think it's a good step. How about putting this class in nadia repo?

Comment (calvertdw) [reply ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/graphics/ros2/RDXROS2InteractableSensors.java:1]: Nice idea, done.

**Review by ds58 — APPROVED** [review]

**Pull request merged**